code/modules/research/designs/autolathe/multi-department_designs.dm

```diff
@@ -537,3 +537,16 @@
 		RND_CATEGORY_MODULAR_COMPUTERS + RND_SUBCATEGORY_MODULAR_COMPUTERS_FRAMES,
 	)
 	departmental_flags = DEPARTMENT_BITFLAG_SCIENCE | DEPARTMENT_BITFLAG_ENGINEERING
+
+/datum/design/universal_scanner
+	name = "Universal Scanner"
+	desc = "A utility scanner that fills multiple roles: Exports, sales, vendor pricetags."
```

Comment:
```suggestion
	desc = "A utility scanner that fills multiple roles: exports, sales, and vendor price tags."
```

Reply:
(I cringe at myself saying this)
Exports would still be capitalized